packages/react/src/components/navigationLink/NavigationLink.module.scss

```diff
@@ -1,9 +1,25 @@
-.navigationLink.navigationLink  {
-    --link-visited-color: none;
-    --link-color: var(--color-black-90);
-    text-decoration: none;
-}
-.navigationLink:hover, .navigationLink:focus {
-    text-decoration: underline;
-    text-decoration-color: var(--color-black-90);
+.navigationLink.navigationLink {
+  --link-visited-color: none;
```

Comment:
This is something I like and appreciate very much! (the variables for this component defined here)

Reply:
My man! Up top ✋